dll/directx/wine/ddraw/ddraw.c

```diff
@@ -1149,6 +1150,10 @@ static HRESULT WINAPI ddraw7_SetDisplayMode(IDirectDraw7 *iface, DWORD width, DW
 
     InterlockedCompareExchange(&ddraw->device_state, DDRAW_DEVICE_STATE_NOT_RESTORED, DDRAW_DEVICE_STATE_OK);
 
+    if (ddraw->cooperative_level & DDSCL_EXCLUSIVE)
+       SetWindowPos(ddraw->dest_window, HWND_TOP, 0, 0, width, height, SWP_SHOWWINDOW | SWP_NOACTIVATE);
+       TRACE("DirectDraw window has been resized\n");
+
```

Comment:
Please guard this block by #ifdef __REACTOS__
as ddraw is code from Wine, not ours.

Reply:
The code is taken from Wine, so why guarding it?

Reply:
https://github.com/wine-mirror/wine/blob/master/dlls/ddraw/ddraw.c at line 1170
shows the line InterlockedCompareExchange(&ddraw->device_state, DDRAW_DEVICE_STATE_NOT_RESTORED, DDRAW_DEVICE_STATE_OK);
after the 
if (ddraw->cooperative_level & DDSCL_EXCLUSIVE)

while for us it is the other way round. Why is that? Bug or intended? That is at least a diff

Reply:
![LetsHaveALook](https://user-images.githubusercontent.com/33393466/192089267-28fec688-ba95-4a18-95a5-1f88ccd0b865.PNG)

Reply:
Does your patch also work if you placed it, where Wine placed its corresponding block?